lib/parse-insee-code.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
```

Comment:
Ne pas utiliser le pragma dans du code backend, l'opt-in ES5.1 est inutile. Si c'est là pour la validité, utiliser le switch de ligne de commande `--use_strict` au lancement de `node`.